src/app.js

```diff
@@ -1,21 +1,35 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer((req, res) => {
+  res.setHeader('Content-type', 'application/json');
+
+  const qIndex = req.url.indexOf('?');
+  const parts = req.url.slice(1, qIndex).split('/');
+  const query = {};
+
+  for (const part of req.url.slice(qIndex + 1).split('&')) {
```

Comment:
`url.slice(qIndex + 1).split('&')` - seems like a good idea to make it a variable)